src/main/java/Data/BookHandler.java

```diff
@@ -6,7 +6,7 @@
 import java.util.*;
 
 public class BookHandler {
```

Comment:
Try to extract interface from this class(IntelliJ helps with this task). BookHandler <-- name of interface. This class will change name for BookHandlerImpl

Reply:
Done